### ISSUE TITLE
Oculus

### ISSUE THREAD

**Issue by BeardedBandits** [developer9998/GorillaGameFix#2]
Hello I use monkey mod manager on the oculus desktop app and I am wondering if there is a way to fix the Beplinx for oculus

**Comment by WowItsKaylie** [comment]
I don't think you can even download PCVR Gorilla Tag from the Oculus app. I assume you mean the Oculus option on startup, in which case this process is the same.
